### PR TITLE
TASK: Document width and height property in ImageUri Fusion-prototype

### PR DESCRIPTION
**Pull request by mficzel**
This documents changes that were introduced in Neos 2.0 with revision 75430099fe2dbd3646eb549dac5e9778abf4c82a.

### CODE REVIEW

File: TYPO3.Neos/Documentation/References/NeosTypoScriptReference.rst [hunk 1]
```diff
@@ -716,8 +718,8 @@ Example::
 
 	logoUri = TYPO3.Neos:ImageUri {
 		asset = ${q(node).property('image')}
-		maximumWidth = 100
-		maximumHeight = 100
+		width = 100
+		height = 100
```
Comment (aertmann) [inline TYPO3.Neos/Documentation/References/NeosTypoScriptReference.rst]: no need to change this example, not sure if a cropped image is more common than a scaled one

Comment (mficzel) [inline TYPO3.Neos/Documentation/References/NeosTypoScriptReference.rst]: The allowCropping option that is used in the example makes more sense with an explicitly defined size.

Comment (mficzel) [inline TYPO3.Neos/Documentation/References/NeosTypoScriptReference.rst]: In fact as far as I know cropping cropping needs this since 2.0 to actually work.

Comment (aertmann) [inline TYPO3.Neos/Documentation/References/NeosTypoScriptReference.rst]: good point, leave it as is

**Pull request merged**